packages/@okta/vuepress-site/docs/guides/common-hook-set-up-steps/main/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Common Hook Set-up Steps
+excerpt: A list of set-up steps that are common to all hook implementations.
+layout: Guides
+---
+
+This guide explains common set-up steps when implementing an Okta Event or Inline hook, including using Glitch.com as an example external service, adding basic authentication to the hook calls, JSON body parsing in the external service code, and troubleshooting steps.
+
+---
+
+**Learning outcomes**
+
+* Understand common set up steps for implementing an Okta Event or Inline Hook
+* Use these steps when running the example hook code in the accompanying guides
+
+**What you need**
+
+* A [Glitch.com](https://glitch.com) project or account.
+* An Okta developer org. [Create an org for free](https://developer.okta.com/signup/).
+
+**Sample code**
+
+* n/a
+
+---
+
+Okta Event and Inline Hooks use outbound calls, which are received and parsed by an external service to implement additional custom functionality for your Okta implementation.
+
+A secure web server and application is a requirement to implement an Okta Event or Inline Hook, and is referred to as the external service in these guides.
+
+For instructional purposes, the following guides for Event Hooks and Inline Hook types use the third-party site [Glitch](https://glitch.com), which functions as an online external service. You can use the example code to quickly implement the hook and preview the functionality.
+
+> **Note:** You can also use your own secure web server to implement the sample code.
+
+See the following section on setting up a Glitch project and adding appropriate code.
+
+## Set up an external service
+
+The Event Hook and Inline Hook examples in this section use Glitch projects to create an external service for use with your Okta org.
+
+[Glitch](https://www.glitch.com) is a browser-based development environment that can build a full-stack web application online. You can use their template applications to implement an external service that receives the outbound calls from Okta orgs.
+
+Start with a new Node.js project built on the Express framework or a Node.js SQLite database application and use the code snippets in the following examples to implement the example hooks. Copy (**Remix on Glitch**) the Glitch projects for each hook in the following sections to have a working code sample.
+
+<StackSelector snippet="setup"/>
+
+## Add Basic Authorization and Body Parsing
+
+The Glitch project templates do not have any authorization or body parsing code. To include this content:
+
+* Add the Body Parser and Basic Auth `npm` packages to your Glitch project
+* Add the code snippet below
+
+If you Remix a Glitch Inline Hook project, the packages and code are already included.
```

Comment:
Should Remix a Glitch Inline Hook project have caps? If so, should it be in quotes or something? It reads kind of funny this way...
```suggestion
If you Remix a Glitch Inline Hook project, the packages and code are already included.
```

Reply:
Good point, I believe I was trying to use the Glitch UI terminology. I'll lower case the "Remix" ... everything else should stay capitalized.